homework/hackyourtemperature/server.js

```diff
@@ -0,0 +1,27 @@
+import express from "express";
+
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+app.use(express.json());
+
+const response = { message: "" };
```

Comment:
Nice idea for a reusable response! Normally what we do is something like this:

```suggestion
const response = (message) => {
  return { message };
};
```

This way there are less side-effects and we are creating a new object each time to return to the user.

Reply:
Thanks for the feedback. Yes, it definitely must be a function instead of just an object there. I implemented that in the next week's version.